src/core/icons/fileIcons.ts

```diff
@@ -2947,5 +2947,9 @@ export const fileIcons: FileIcons = {
       name: 'wrangler',
       fileNames: ['wrangler.toml'],
     },
+    {
+      name: 'mail',
+      fileExtensions: ['eml', 'msg'],
```

Comment:
What about adding this to the existing email icon? Would be the easiest way. I don't see why we would require another color here.

Reply:
I didn't know we had this icon, I'll look into it later.

Reply:
> What about adding this to the existing email icon? Would be the easiest way. I don't see why we would require another color here.

It is better to use a different color to distinguish them from mailmap, anyway they are different file types

Reply:
Also in our project, for example, the color blue is associated with documentation and markdown

The "messages" folder is yellow, so why not use yellow for emails as a message type?

Reply:
People might got used to the blue color already. The blue color looks good for many icons, that's why there was a blue color for the mail icon. Even the Outlook logo or the thunderbird logo are blueish, so I don't see a need for a change here.

Reply:
Ok